### PR TITLE
Add some additional help on google assistant trouble shooting

### PR DESCRIPTION
**Pull request by elupus**
## Proposed change

Add some additional trouble shooting information for google assistant.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/google_assistant.markdown [hunk 1]
```diff
@@ -240,7 +242,20 @@ The `request_sync` service may fail with a 404 if the `project_id` of the HomeGr
   4. Generate a new API key.
   5. Again, create a new project in the [Actions on Google console](https://console.actions.google.com/). Described above. But at the step 'Build under the Actions SDK box' choose your newly created project. By this, they share the same `project_id`.
 
-### Troubleshooting with NGINX
+#### 403 errors on request sync
+
+The `request_sync` service may fail with a 403 if the if the HomeGraph API is not enabled. Go to [Google API Console](https://console.cloud.google.com/apis/api/homegraph.googleapis.com/overview) and verify that HomeGraph API is enabled for your project
```
Comment (frenck) [inline source/_integrations/google_assistant.markdown]:
```suggestion
The `request_sync` service may fail with a 403 if the HomeGraph API is not enabled. Go to [Google API Console](https://console.cloud.google.com/apis/api/homegraph.googleapis.com/overview) and verify that HomeGraph API is enabled for your project.
```

File: source/_integrations/google_assistant.markdown [hunk 2]
```diff
@@ -240,7 +242,20 @@ The `request_sync` service may fail with a 404 if the `project_id` of the HomeGr
...
+
+#### 404 errors on report state
+
+If you receove 404 errors linked to report state in your log, home assisting is reporting state for entites that where never synced to google. Ask your google home to `Sync my devices` or run the service `google_assistant.request_sync`
+
```
Comment (frenck) [inline source/_integrations/google_assistant.markdown]:
```suggestion
If you receive 404 errors linked to reporting state in your log, Home Assistant is reporting state for entities that were never synced to google. Ask your Google Home to `Sync my devices` or run the service `google_assistant.request_sync`.
```

File: source/_integrations/google_assistant.markdown [hunk 3]
```diff
@@ -240,7 +242,20 @@ The `request_sync` service may fail with a 404 if the `project_id` of the HomeGr
...
+
+#### Error during linking: "Could not update the setting. Please check your connection"
+
+Your fullfillment url may be invalid or unreachable. Recheck the `Fulfillment URL` as specified in [Manual Setup](#manual-setup) and verify that it's publically reachable.
```
Comment (frenck) [inline source/_integrations/google_assistant.markdown]:
```suggestion
Your fulfillment URL may be invalid or unreachable. Recheck the `Fulfillment URL` as specified in [Manual Setup](#manual-setup) and verify that it's publically reachable.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by elupus** [comment]
Christ, i'm spelling like a... Thank you for to corrections.

**Review by frenck — APPROVED** [review]
Thanks, @elupus 👍

**Pull request merged**